controlled-vocabulary/src/main/java/org/uniprot/core/cv/xdb/UniProtXDbTypeDetail.java

```diff
@@ -6,16 +6,18 @@
 import java.util.Objects;
 
 public class UniProtXDbTypeDetail implements Serializable {
+
     private static final long serialVersionUID = 8751881513996820892L;
+    public static String EXPLICIT = "Explicit";
```

Comment:
make it final? otherwise, don't capitalise the var name, for consistency with other static final vars.